governance/votes/Executive vote - December 10, 2021.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Template - [Executive Vote] Parameter Changes, Switching MKR Vesting Source - December 10, 2021
+summary: Changes to wstETH-A and MATIC-A Parameters, replacing current MKR vesting streams with streams funded by the Protocol Treasury MKR.
```

Comment:
(I think we don't usually use a period at the end here. Maybe there's no real reason for that, but I thought I'd mention it just in case it'd break the voting portal or something).

Reply:
So we usually do, last week'd didn't I believe because Long was trying out the hyphen format which I was fine with but didn't seem necessary for a shorter explanation